### DISCUSSION TITLE
Nuxt /server routes 404 on Cloudflare Pages

### DISCUSSION THREAD

**Discussion by jasonhibbs** [nitrojs/nitro#2000] [General]
I'm evaluating hosting and Nuxt/Nitro says Cloudflare Pages is a zero config solution, but all my /api routes 404.

https://wurdit.pages.dev/

Is this expected and intended for static sites only?

I'm currently running the same deploy fine on Netlify at https://wurd.it

**Comment by DavidDeSloovere** [comment]
Our app runs fine on CF pages.
Using the Github/git integration to have CF build and upload.

**Reply by jasonhibbs** [reply]
Thanks for sharing!

My server routes are a proxy to an external API, are you also calling APIs?

**Reply by DavidDeSloovere** [reply]
Yes, we have our own server routes and APIs in Nitro that call into another GraphQL endpoint externally. Works fine.

Not using the builtin proxy option however. 

In CF dashboard, or with wrangler, you can connect to streaming logs. Those might help you? 

Have you tried creating a minimal Nuxt site deployed on CF?
